daml-assistant/daml-helper/src/DamlHelper.hs

```diff
@@ -210,11 +210,11 @@ runStart = withProjectRoot $ \_ -> do
     mbScenario :: Maybe String <-
         requiredE "Failed to parse scenario" $
         queryProjectConfig ["scenario"] projectConfig
-    let darName = projectName <> ".dar"
+    let darPath = "dist" </> projectName <> ".dar"
     assistant <- getDamlAssistant
-    callCommand (unwords $ assistant : ["build", "-o", darName])
+    callCommand (unwords $ assistant : ["build", "-o", darPath])
```

Comment:
This is the default output path of `build` even without a `-o` argument. Is there an easy way to just use the same path as `build`? e.g. if we want to make the build location configurable in the future, we have to change it here as well.

Reply:
I guess we could move it to `daml-project-config` but that also feels like the wrong place for it. Not passing `-o` doesn’t really help since we need to pass a path to sandbox and navigator so we still have to specify an explicit path. I’d say we probably change this sufficiently rarely that I don’t see this being a big problem.